docs/reference/cluster/tasks.asciidoc

```diff
@@ -11,11 +11,12 @@ executing on one or more nodes in the cluster.
 
 [source,js]
 --------------------------------------------------
-GET /_tasks <1>
-GET /_tasks?nodes=nodeId1,nodeId2 <2>
-GET /_tasks?nodes=nodeId1,nodeId2&actions=cluster:* <3>
+GET _tasks <1>
+GET _tasks?nodes=nodeId1,nodeId2 <2>
+GET _tasks?nodes=nodeId1,nodeId2&actions=cluster:* <3>
 --------------------------------------------------
 // AUTOSENSE
+// TEST
```

Comment:
I think I should be able to drop these in most cases and just make tests out of `// AUTOSENSE`. I'm not sure if that is something that should wait for a second PR though. Just testing some stuff is nice. Maybe it is worth getting that in and then doing something to test *all* stuff.